tests/test_fips_iut.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+from unittest import TestCase
+
+from sec_certs.dataset.fips_iut import IUTDataset
+from sec_certs.sample.fips_iut import IUTSnapshot
+
+
+class TestFIPSIUT(TestCase):
+    def setUp(self):
+        self.test_data_dir = Path(__file__).parent / "data" / "test_fips_iut"
+        self.test_data_dump = self.test_data_dir / "fips_iut_2020-10-25T06+01:00.html"
```

Comment:
This could probably be defined in `setUpClass` classmethod? As of now, this method is invoked before each test. Also, variables could be declared as `ClassVar`s.